### PR TITLE
Issue/13 dashboard component

### PR DESCRIPTION
**Pull request by hithwen**
This is not ready to merge but is better to start discussing the changes.
I cannot get the css styles to load (both flexbox and font awesome)

### CODE REVIEW

**Comment by brauliodiez** [comment]
@nasdan can you take a look on this? If you jump into máster (and removí gracias node modules) It seems.you get bootstrap errores and require not found

File: package.json [hunk 1]
```diff
@@ -86,6 +86,8 @@
     "@types/redux-thunk": "^2.1.31",
     "@types/webpack-env": "^1.12.1",
     "bootstrap": "^3.3.7",
+    "flexbox": "0.0.3",
```
Comment (Nasdan) [inline package.json]: Why do you install this package?

File: src/common/components/dashboard/dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import {DashboardItemComponent, IDashboardItem} from "./dashboard-item";
+
```
Comment (Nasdan) [inline src/common/components/dashboard/dashboard.tsx]: You can declare an interface here to be more readable:

```javascript
interface Props {
 title: string;
 items: IDashboardItem[]
}
```

File: webpack.config.js [hunk 3]
```diff
@@ -67,7 +67,9 @@ module.exports = {
       //NOTE: Bootstrap css configuration
       {
         test: /\.css$/,
-        include: /node_modules\\bootstrap/,
+        include: [/node_modules\\bootstrap/,
```
Comment (Nasdan) [inline webpack.config.js]: include is a property that expect a regular expression

File: webpack.config.js [hunk 4]
```diff
@@ -67,7 +67,9 @@ module.exports = {
...
+          /src\\common\\components/,
```
Comment (Nasdan) [inline webpack.config.js]: You don't need to add src/common/components because next loader is loading .scss files from src

**Comment by Nasdan** [comment]
Hi @hithwen, I'm going to continue with this issue #13

**Pull request closed without merging**